pixels-common/src/main/java/io/pixelsdb/pixels/common/metadata/MetadataService.java

```diff
@@ -294,6 +294,34 @@ public Table getTable(String schemaName, String tableName, boolean withLayouts)
         return table;
     }
 
+    public Table getTable(long tableId) throws MetadataException
```

Comment:
Method name should be the same as the corresponding method in MetadataServiceImpl.